### PR TITLE
[camera] Fix ImageStream ImageFormatGroup is Ignored in iOS

### PR DESCRIPTION
**Pull request by zuvola**
Fixes [flutter/flutter#93842](https://github.com/flutter/flutter/issues/93842)

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [relevant style guides] and ran [the auto-formatter]. (Unlike the flutter/flutter repo, the flutter/plugins repo does use `dart format`.)
- [x] I signed the [CLA].
- [x] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. `[shared_preferences]`
- [x] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy], or this PR is [exempt from version changes].
- [x] I updated `CHANGELOG.md` to add a description of the change, [following repository CHANGELOG style].
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[relevant style guides]: https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md#style
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[exempt from version changes]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#version-and-changelog-updates
[following repository CHANGELOG style]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style
[the auto-formatter]: https://github.com/flutter/plugins/blob/master/script/tool/README.md#format-code
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests

### CODE REVIEW

File: packages/camera/camera/ios/Classes/CameraPlugin.m [hunk 1]
```diff
@@ -371,6 +370,8 @@ - (instancetype)initWithCameraName:(NSString *)cameraName
   _focusMode = FocusModeAuto;
   _lockedCaptureOrientation = UIDeviceOrientationUnknown;
   _deviceOrientation = orientation;
+  // Format used for video and image streaming.
```
Comment (stuartmorgan) [inline packages/camera/camera/ios/Classes/CameraPlugin.m:373]: Comments about the purpose of a variable should be on the declaration (in this case, the property) not the assignment.

**Review by stuartmorgan — COMMENTED** [review]
Thanks for the contribution!

This change looks like it makes sense, but I'm having a hard time reconciling the change with the description of the issue (and the test). It looks like it wasn't being ignored, it was being tracked statically rather than per-instance—which I would expect to result in issues only if there were multiple instances of the camera. Am I missing something important about the old implementation?

**Comment by zuvola** [comment]
Thank you for the review.
The previous implementation only changed the static variable ’videoFormat’ during initialize, but did not assign a value to _captureVideoOutput.videoSettings.
Therefore, the actual captured image would be the default BGRA, and the values I set would be ignored.

File: packages/camera/camera/ios/Classes/CameraPlugin.m (hunk 1 above)
Comment (zuvola) [reply packages/camera/camera/ios/Classes/CameraPlugin.m]: I fixed it.

**Review by stuartmorgan — APPROVED** [review]
Thanks for the explanation, LGTM!

@cyanglaz for secondary review.

File: packages/camera/camera/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+##  0.9.4+6
+
+* Fixed an bug where ImageFormatGroup was ignored in startImageStream on iOS.
```
Comment (bparrishMines) [inline packages/camera/camera/CHANGELOG.md:3]: nit: 
```suggestion
* Fixed a bug where ImageFormatGroup was ignored in `startImageStream` on iOS.
```

**Review by bparrishMines — CHANGES_REQUESTED** [review]
LGTM with a small nit pick. The version will also need to be bumped one more time.

**Comment by stuartmorgan** [comment]
@bparrishMines I resolved the conflicts and applied your suggestion to the CHANGELOG

**Review by bparrishMines — APPROVED** [review]
LGTM

**Comment by bparrishMines** [comment]
It looks like setting the default email for an organization is still an issue: https://github.com/todogroup/gh-issues/issues/32. I'll can fix the CLA manually.

**Comment by bparrishMines** [comment]
@googlebot I signed it!

**Comment by bparrishMines** [comment]
I signed it!

**Pull request merged**